backend/campaigns/admin.py

```
@@ -0,0 +1,169 @@
+    def human_hits(self, obj):
+        return obj.human_hit_count
+
+    @admin.display(ordering='bot_hit_count', description='Hits (bots)')
+    def bot_hits(self, obj):
+        return obj.bot_hit_count
+
+    @admin.display(ordering='signup_count', description='Signups')
+    def signups(self, obj):
+        return obj.signup_count
+
+    def get_readonly_fields(self, request, obj=None):
```

Ruff (0.16.0)

[warning] Unused method argument: `request`

(ARG002)